src/main/java/org/jenkinsci/plugins/github/webhook/subscriber/PingGHEventSubscriber.java

```diff
@@ -59,18 +61,23 @@ protected boolean isApplicable(Item project) {
      */
     @Override
     protected void onEvent(GHEvent event, String payload) {
```

Comment:
In just you can trust to any underlying extension as it may throw NPE, so catching throwables and reporting is standard save ass pattern. If exception throws meaningful information then additional exception steps doesn't add anything useful for debugging.